mava/systems/tf/mappo/builder.py

```diff
@@ -75,13 +76,19 @@ class MAPPOConfig:
     environment_spec: specs.EnvironmentSpec
     policy_optimizer: Union[snt.Optimizer, Dict[str, snt.Optimizer]]
     critic_optimizer: snt.Optimizer
+    num_executors: int
     agent_net_keys: Dict[str, str]
+    trainer_networks: Dict[str, List]
+    table_network_config: Dict[str, List]
+    network_sampling_setup: List
+    net_keys_to_ids: Dict[str, int]
+    unique_net_keys: List[str]
     checkpoint_minute_interval: int
     sequence_length: int = 10
     sequence_period: int = 5
     discount: float = 0.99
     lambda_gae: float = 0.95
-    max_queue_size: int = 100_000
+    max_queue_size: int = 1000
```

Comment:
Because of the exact reason you pointed out in your link :smile: `This (large queue size) will not work for distributed setup
    # if learner is slower than actors (the queue will fill up and the learner
    # will be offpolicy).`